webpack/envs/novoDevelopmentConfig.js

```diff
@@ -154,6 +154,7 @@ export const novoDevelopmentConfig = {
         NODE_ENV: JSON.stringify(env.nodeEnv),
       },
     }),
+    new webpack.HotModuleReplacementPlugin(),
```

Comment:
Did this resolve some of the HMR issues?

Reply:
I'm hoping, also very confused that this change is in this PR.

Reply:
Nah, i was just seeing if there might be some loading order strangeness going on, no different.